src/app/views/provisioning/subscriptions.less

```diff
@@ -1,5 +1,10 @@
 #provisioning-subscriptions {
+ width: 90vw;
+ position: relative;
+ margin-left: -49vw;
+ left: 55%;
```

Comment:
I'm not sure about the 
```
margin-left: -49vw;
left: 55%;
```
It looks hacky to me. `width: 90vw;` is 90% of the viewport width, I think you can have a `margin-left` the size of the left side menu (I think there is a variable).

Reply:
There is no variable for the width of the sidebar menu. If there was a variable, using margin-left would not work due to its structure. 
This ID is wrapped within a container, which is wrapped within a container-fluid, so setting a margin-left would not work since it would be using the bootstrap container(ui-view) as a reference. I have used VW to be able to ignore its parent container and set margin-left and left is the only way to centre it I think.

I have created this codepen to be more clear on my explanation.
https://codepen.io/heDu/pen/mBRzMd